docs/conf.py

```diff
@@ -59,6 +60,7 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.napoleon',
     'sphinx.ext.todo',
+    'ext'
```

Comment:
We should give this extension a better name.

Reply:
Fixed.